src/etc/inc/system.inc

```diff
@@ -1365,6 +1343,49 @@ function system_webgui_start() {
 	return $res;
 }
 
+function get_dns_nameservers() {
+	global $config;
+	
+	$dns_nameserver_number = 0;
```

Comment:
This variable is declared but never used.

Reply:
I'm a c programmer at heart. I was originally going to use a for loop and I don't know about php's variable declarations greatly. I learned about foreach a little later. deleting.